providers/install.rb

```diff
@@ -18,14 +18,23 @@
 #
 
 action :run do
-  @tarball = "#{new_resource.base_name}#{new_resource.version}.#{new_resource.artifact_type}"
+  # Package install
+  if node['redisio']['package_install']
+    package "redisio_package_name" do
+      package_name node['redisio']['package_name']
+      action :install
```

Comment:
Hello @deimosfr, Suggestion to consider specifying the version attribute for this package resource; that's consistent with ```default['redisio']['version']``` Such that the installation is version locked and consistent with a given version, otherwise it will install whichever is latest package available in the repo at that point of time.